docs/freeze.md

```diff
@@ -0,0 +1,35 @@
+# Guest file system freeze
+
+Kubevirt provides a way to freeze/thaw the guest filesystem. That can be used to ensure the filesystem consistency
+when making a backup or a VM snapshot. The freeze requires a running qemu-guest-agent in the guest VM.
+
+Note: To check if your vm has a qemu-guest-agent check for 'AgentConnected' in the vm status.
+
+## freeze/unfreeze subresource API
+
+VirtualMachineInstance API implements a subresource for a freeze and unfreeze commands.
+When called it sends respective freeze/unfreeze commands to the virt-launcher.
+
+The freeze/unfreeze can be accessed using HTTP PUT command on a vmi REST API or a by a Kubevirt client-go.
+API subresource is located under a `virtualmachineinstances` resource
+(example uri for VM `example-vm` in namespace `demo` is `apis/subresources.kubevirt.io/v1alpha3/namespaces/demo/virtualmachineinstances/example-vm/freeze`).
+
+It can also be access through kubevirt/client-go. It is available on VirtualMachineInstanceInterface defined in `kubevirt.io/client-go/kubecli/kubevirt.go`.
+
+```
+virtClient.VirtualMachineInstance(namespace).Freeze(vmiName, unfreezeTimeout)
+virtClient.VirtualMachineInstance(namespace).Unfreeze(vmiName)
+```
+
+The freeze/unfreeze subresources are internally used by the VM VirtualMachineSnapshot/VirtualMachineRestore API
```

Comment:
it is used only in the VirtualMachineSnapshot it is not used in the restore

Reply:
right, fixed